test/remoteDebug/getRemoteDebugLanguage.test.ts

```diff
@@ -0,0 +1,68 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as assert from 'assert';
+import { RemoteDebugLanguage } from 'vscode-azureappservice';
+import { IActionContext } from 'vscode-azureextensionui';
+import { getRemoteDebugLanguage } from '../../extension.bundle';
+
+suite('getRemoteDebugLanguage', () => {

```

Comment:
Could you update the tests to include getRemoteDebugLanguage with the Python flag enabled?  I'm adding this helper function that should aid in that https://github.com/microsoft/vscode-azureappservice/pull/1149